modules/nf-core/modules/bcftools/stats/main.nf

```diff
@@ -9,6 +9,8 @@ process BCFTOOLS_STATS {
 
     input:
     tuple val(meta), path(vcf)
```

Comment:
```suggestion
    tuple val(meta), path(vcf), path(vcf_tbi)
```

Reply:
Let's keep the naming as in the other modules (i.e. `bcftools/consensus`). 
When everything is working please update the `bcftools/stats` module upstream.

Reply:
I had also thought about trying out sth like this to find the right index file in the script section:
`INDEX=`find -L ./ -name "${prefix}.vcf.tbi"`` and then passing the `\$INDEX \` to the `bcftools/stats` command, but i could not get it to run due to nextflow/bash problems.

Reply:
Hi @SusiJo and @maxulysse : Sorry about this mess and thanks a lot for all your comments and links to previous discussions. I'll take a careful look at it all.

Reply:
@maxulysse : I'm including the tbi-file in the input of the module `bcftools/stats` because, without it, the following test 
```
nextflow run main.nf -profile test,tools_tumoronly,singularity --tools manta
```
resulted in the following error:
```
  Process `NFCORE_SAREK:SAREK:VCF_QC:BCFTOOLS_STATS (sample2)` terminated with an error exit status (255)

Command executed:

  bcftools stats \
       \
      --regions-file multi_intervals.bed \
      sample2.manta.tumor_sv.vcf.gz > sample2.manta.tumor_sv.bcftools_stats.txt

  cat <<-END_VERSIONS > versions.yml
  "NFCORE_SAREK:SAREK:VCF_QC:BCFTOOLS_STATS":
      bcftools: $(bcftools --version 2>&1 | head -n1 | sed 's/^.*bcftools //; s/ .*$//')
  END_VERSIONS

Command exit status:
  255

Command output:
  (empty)

Command error:
  [E::idx_find_and_load] Could not retrieve index file for 'sample2.manta.tumor_sv.vcf.gz'
  Failed to read from sample2.manta.tumor_sv.vcf.gz: could not load index

Work dir:
  /home/aspe/dev/fork_sarek/work/9b/0c3268ad4e17ee489871c6849292ff
  ```